library/cwm/src/modules/CWM.rb

```diff
@@ -135,6 +139,9 @@ def ProcessTerm(t, widgets)
             arg = ProcessTerm(Convert.to_term(arg), widgets)
           end
         elsif Ops.is_string?(arg) # action
+          Builtins.y2error("find string #{arg} in term #{t.inspect} which is unknown.") unless widgets[arg]
```

Comment:
OK, I want to have explicit error when there is a string and there are no widget defined with given id. CWM continue with replacing it with `VBox()` which is extremelly annoying for debug. So I want to have explicit error in log.

So message trying to say. "I found in StringTerm string which cannot be associated to any defined widget."